ckanext/switzerland/tests/test_format_utils.py

```diff
@@ -219,5 +203,14 @@ def test_prepare_resource_format(self):
             'format': 'comma ...'
         }
         resource_with_comma_format_cleaned = ogdch_format_utils.prepare_resource_format(
-            resource_with_comma_format.copy(), format_mapping=ogdch_mixin.format_mapping)
+            resource_with_comma_format.copy())
         self.assertEquals('CSV', resource_with_comma_format_cleaned['format'])
+
```

Comment:
A new test case has been added below to test that already mapped format wont be mapped again.